packages/klevu-ui/src/components/klevu-recommendations/klevu-recommendations.tsx

```diff
@@ -72,19 +74,23 @@ export class KlevuRecommendations {
       throw new Error("recommendationId is required")
     }
 
-    const res = await KlevuFetch(
-      kmcRecommendation(
-        this.recommendationId,
-        {
-          id: "recommendation",
-          cartProductIds: this.cartProductIds,
-          categoryPath: this.categoryPath,
-          currentProductId: this.currentProductId,
-          itemGroupId: this.itemGroupId,
-        },
-        sendRecommendationViewEvent(this.recommendationTitle)
-      )
-    )
+    let fetchOptions: KlevuFetchFunctionReturnValue = await kmcRecommendation(this.recommendationId, {
+      id: "recommendation",
+      cartProductIds: this.cartProductIds,
+      categoryPath: this.categoryPath,
+      currentProductId: this.currentProductId,
+      itemGroupId: this.itemGroupId,
+    })
+
+    const settings = getKMCSettings()
+    if (settings) {
```

Comment:
Why do you fetch settings if you do not use them for anything?

Reply:
removed